lib/beaker/options/presets.rb

```diff
@@ -38,6 +38,7 @@ class Presets
         :test_tag_or          => ['BEAKER_TEST_TAG_OR'],
         :test_tag_exclude     => ['BEAKER_EXCLUDE_TAG', 'BEAKER_TEST_TAG_EXCLUDE'],
         :run_in_parallel      => ['BEAKER_RUN_IN_PARALLEL'],
+        :use_fog_credentials  => 'true',
```

Comment:
@cdenneen this should be added to the list of options in the `presets` method down below in this file. Additionally, it should be just `true`, not `'true'`. 

Thanks!

Reply:
Done

Reply:
@cdenneen it looks like it's still a string here, is there a reason for that/

Reply:
@kevpl ENV spec requires strings (doesn't work with bool).
The real preset is a bool:

https://github.com/cdenneen/beaker/blob/1b7ff68dab806dce9ebfce928e3e42f2dd8f1681/lib/beaker/options/presets.rb#L197

Reply:
would it be better if I just removed it from there or should we be adding a BEAKER_USE_FOG ?